README.rst

```diff
@@ -1,6 +1,9 @@
-.. image:: https://img.shields.io/travis/Yelp/bravado.svg
+.. image:: https://img.shields.io/travis/Yelp/bravado/master.svg?label=Linux+Build
   :target: https://travis-ci.org/Yelp/bravado?branch=master
 
+.. image:: https://img.shields.io/appveyor/ci/Yelp/bravado/master.svg?label=Windows+Build
+  :target: https://ci.appveyor.com/project/Yelp/bravado/
```

Comment:
That link is not working, should we maybe remove the badge for now?

Reply:
You're right ... I forgot to push the change :)